src/EnergyPlus/WindowManager.cc

```diff
@@ -6990,17 +6988,18 @@ namespace WindowManager {
         Real64 rIn = 1.0 / (hInRad + state.dataWindowManager->hcin);
         Real64 Rbare = 0;
 
-        auto const SELECT_CASE_var(state.dataWindowManager->ngllayer);
+        switch (state.dataWindowManager->ngllayer) {
 
-        if (SELECT_CASE_var == 1) {
+        case 1:
```

Comment:
These represent the simple, integer, numbers of layers.  I am actually inclined to leave it as plain integers here, but I could make a set of dummy enums: `{One = 1, Two = 2, Three = 3, etc.}` if desired.

Reply:
I agree, please add a comment above the `switch` to the effect that this is what is happening, so that the next person doesn't have to figure it out again.